pages.hi/common/git-status.md

```diff
@@ -0,0 +1,20 @@
+# git status
+
+> उन पथों को प्रदर्शित करता है जिनमें अनुक्रमणिका फ़ाइल और वर्तमान हेड कमिट के बीच अंतर होता है ।
```

Comment:
Please sync this command description, the 2nd line from the main page is missing.

Reply:
hello @navarroaxel , the description from main (English) and translated description resembles almost the same meaning , but no worries i will make them sync 1:1